src/app/api/_api-services/onchain_db_service/subsquid_service/index.ts

```
@@ -761,22 +763,44 @@ export class SubsquidService extends SubsquidUtils {
+			const existing = delegateMap.get(delegate);
+
+			if (existing) {
+				existing.push(delegation);
+			} else {
+				delegateMap.set(delegate, [delegation]);
+			}
+		});
+
+		// Calculate results for each delegate - optimized iteration
+		delegateMap.forEach((delegations, delegate) => {
+			const maxDelegated = this.calculateMaxTrackVotingPower(delegations);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)